tests/periph_i2c/app.config.test

```diff
@@ -0,0 +1,15 @@
+# this file enables modules defined in Kconfig. Do not use this file for
+# application configuration. This is only needed during migration.
+
+# Enable I2C peripheral driver
+CONFIG_MODULE_PERIPH_I2C=y
+
+# Enable I2C scan
+CONFIG_MODULE_I2C_SCAN=y
+
+# Enable xtimer to wake up periodically
+CONFIG_MODULE_ZTIMER=y
+CONFIG_MODULE_ZTIMER_MSEC=y
```

Comment:
Are these config parameters necessary?

Reply:
Uhmmm, not really, it's only a copy pasta issue xD